### PR TITLE
Add cleanup job logging

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. Thx.

File: app/jobs/cleanup.js [hunk 1]
```diff
@@ -16,51 +16,54 @@ const CleanupJob = function(config, headerDB, metadataDB, logger) {
...
         fileSystem.isUnused(filePath, (isUnused) => {
           if (isUnused) {
 
             fileSystem.delete(filePath, (err) => {
               if (err) {
-                self.logger.error(err, filePath);
+                this.logger.error(err, filePath);
               } else {
-                self.logger.info("File deleted", filePath);
+                this.logger.info("File deleted", filePath);
```
Comment (rodrigopavezi) [inline app/jobs/cleanup.js]: @donnapep curious, couldn't we have tests for the logging of this and the others about headers and metadata? cheers

Comment (donnapep) [inline app/jobs/cleanup.js]: @rodrigopavezi I added tests for the other logs, which helped me to realize that this wasn't implemented correctly. Please review latest commits. Thx!

Comment (donnapep) [inline app/jobs/cleanup.js]: @rodrigopavezi Also, `logger.info` only takes a single parameter so I removed the unnecessary one.

**Pull request merged**